riot/riot.py

```diff
@@ -241,6 +261,17 @@ def venv_path(self) -> str:
         venv_postfix = "_".join([f"{n}{rmchars('<=>.,', v)}" for n, v in self.pkgs])
         return f"{base_path}_{venv_postfix}"
 
+    def site_packages_path(self) -> str:
```

Comment:
This is the same method as defined above? Surprised linting doesn't catch this lol

Reply:
It is on a different class?

Reply:
oh... I missed that but my concern was the duplicated code (the bodies of the two methods are essentially the same)

Reply:
_almost_ yeah, I'll see if I can refactor a bit with the other suggested change